From 7e07485fb23d53036bdc8990bcde45678cb8b341 Mon Sep 17 00:00:00 2001
From: Daniel Borkmann <dborkman@redhat.com>
Date: Tue, 22 Jan 2013 11:38:03 +0100
Subject: [PATCH 36/53] nagle_snd: remove variable with assignment-only
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fixes:

  nagle_snd.c: In function ‘main’:
  nagle_snd.c:96:13: warning: variable ‘bytes_sent’ set but not used [-Wunused-but-set-variable]

It is not needed, since test_sendmsg would fail anyway if the return
value matches not the expected one within the wrapper function.

Signed-off-by: Daniel Borkmann <dborkman@redhat.com>
---
 src/apps/nagle_snd.c | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/src/apps/nagle_snd.c b/src/apps/nagle_snd.c
index 825eb28..4c41e63 100644
--- a/src/apps/nagle_snd.c
+++ b/src/apps/nagle_snd.c
@@ -93,7 +93,7 @@ main(int argc, char *argv[])
 	struct sctp_sndrcvinfo *sinfo;
         struct iovec out_iov;
         char *message;
-	int error, bytes_sent, pf_class;
+	int error, pf_class;
         sctp_assoc_t associd;
 	uint32_t ppid;
 	uint32_t stream;
@@ -287,7 +287,7 @@ main(int argc, char *argv[])
 	       remote_port);
 
         /* Send the first message.  This will create the association.  */
-        bytes_sent = test_sendmsg(sk, &outmessage, 0, size+1);
+        test_sendmsg(sk, &outmessage, 0, size+1);
 
         memset(&inmessage, 0, sizeof(inmessage));	
 	big_buffer = test_malloc(REALLY_BIG);
@@ -346,7 +346,8 @@ main(int argc, char *argv[])
 
 		outmessage.msg_iov->iov_base = message;
 	        outmessage.msg_iov->iov_len = size + 1;
-		bytes_sent = test_sendmsg(sk, &outmessage, 0, size+1);
+
+		test_sendmsg(sk, &outmessage, 0, size+1);
 	}
         
         printf("\n\n\t\tComplete all the data sendings to receiver...\n\n\n");
-- 
1.7.11.7

